values.yaml

```diff
@@ -289,3 +293,11 @@ logging:
   # logging.json -- Where to send logs that are formatted as JSON. Set to an
   # empty string to disable.
   json: ""
+
+  ## Splunk logging configuration -- Coder can send logs directly to
```

Comment:
These are intended to be 
```
<yaml key> -- comment string
```
where the yaml key matches up with the object path of what it's documenting